code/datums/traits/neutral.dm

```diff
@@ -224,3 +224,39 @@
 	SIGNAL_HANDLER
 
 	SEND_SIGNAL(quirk_holder, COMSIG_ADD_MOOD_EVENT, "bad_hair_day", /datum/mood_event/bald)
+
+/datum/quirk/prosthetic_limb
+	name = "Prosthetic Limb"
+	desc = "An accident caused you to lose one of your limbs. Because of this, you now have a prosthetic or amputated limb! (Go to appearance to customize)"
+	value = 0
+	medical_record_text = "During physical examination, patient was found to have a missing limb."
+
+/datum/quirk/prosthetic_limb/on_spawn()
+	var/mob/living/carbon/human/H = quirk_holder
+	var/limb_list = H.client?.prefs.prosthetic_limbs
+	for(var/L in limb_list)
+		var/obj/item/bodypart/old_part = H.get_bodypart(L)
+		switch(limb_list[L])
+			if("normal")
+				continue
+			if("amputated")
+				old_part.drop_limb()
+				qdel(old_part)
+			if("prosthetic")
+				var/obj/item/bodypart/prosthetic
+				var/prosthetic_type = "surplus"
+				switch(H.client?.prefs.pref_species.type)  // Can be expanded as more species prosthetics are added
+					if(/datum/species/kepori)
+						prosthetic_type = "kepori"
+				var/typepath = text2path("/obj/item/bodypart/[L]/robot/[prosthetic_type]") // Dynamically makes the path so I don't have to type this shit out
+				if(!ispath(typepath))
+					to_chat(H, "<span class='warning'>Problem initializing [L] prosthetic for species [H.client?.prefs.pref_species], it will be a normal limb. Make a bug report on github!</span>")
+					continue
```

Comment:
```suggestion
				var/typepath 
				if(ishumanbasic(H)
					typepath = text2path(/obj/item/bodypart/[L]/robot
				else
					typepath = text2path("/obj/item/bodypart/[L]/robot/[H.dna.species.id]") // Dynamically makes the path so I don't have to type this shit out
				if(!ispath(typepath))
					to_chat(H, "<span class='warning'>Problem initializing [L] prosthetic for species [H.dna.species], it will be a normal limb. Make a bug report on github!</span>")
					continue
```

Reply:
this throws an error on any species but human, give humans cyborg limbs instead of prosthetic ones, and won't work as the DNA is not initialized yet (I only know because I made the last error earlier, and had to change it out for pref_species)
also, I don't believe any of the other species have limb types yet, so this would give everyone but humans and kepori the error message and no prosthetic

Reply:
just because they dont have any yet isn't an excuse to not easily make it extensible now.

Also why the FUCK do quirks set up before dna init? Who coded this garbage?

Reply:
because it won't work if you make it easily extensible now.... the code you have will make an invalid typepath for anything but humans so no other species can have prosthetic limbs

Reply:
does ispath not fail if the typepath is invalid? Thats crung. Add a var on `/datum/species` for "has unique prosthesis"

Reply:
it does fail, and it gives them a normal, non-prosthetic limb. I'll add that var and implement this code